common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -503,15 +502,30 @@ def dispatch_to_register():
         """Redirects to the registration page."""
         return redirect(AUTH_DISPATCH_URLS[AUTH_ENTRY_REGISTER])
 
+    def get_provider():
+        """
+        Get's third-party provider for request
+        """
+        return provider.Registry.get_from_pipeline({'backend': backend.name, 'kwargs': kwargs})
+
+    def should_autoprovision_account():
+        current_provider = get_provider()
+        return current_provider and current_provider.autoprovision_account
+
     def should_force_account_creation():
         """ For some third party providers, we auto-create user accounts """
```

Comment:
This docstring should be updated and this method should probably be renamed to `autosubmit_registration_form` or something... it's confusing to tell the difference between `should_autoprovision_account` and `should_force_account_creation`.